src/account/azext_account/vendored_sdks/subscription/models/_models.py

```diff
@@ -316,24 +316,28 @@ class PutAliasRequestProperties(msrest.serialization.Model):
     :param subscription_id: This parameter can be used to create alias for existing subscription
      Id.
     :type subscription_id: str
+    :param reseller_id: Reseller ID, basically MPN Id.
+    :type reseller_id: str
     """
 
     _attribute_map = {
         'display_name': {'key': 'displayName', 'type': 'str'},
         'workload': {'key': 'workload', 'type': 'str'},
         'billing_scope': {'key': 'billingScope', 'type': 'str'},
         'subscription_id': {'key': 'subscriptionId', 'type': 'str'},
+        'reseller_id': {'key': 'resellerId', 'type': 'str'},
     }
 
     def __init__(
         self,
         **kwargs
     ):
         super(PutAliasRequestProperties, self).__init__(**kwargs)
-        self.display_name = kwargs['display_name']
-        self.workload = kwargs['workload']
-        self.billing_scope = kwargs['billing_scope']
+        self.display_name = kwargs.get('display_name', None)
```

Comment:
this looks like manual fixing :) right? will it be overwritten in next generation?

Reply:
The change is auto-generated and caused by a fix in the swagger to mark the property as optional.